book/controller.rst

```diff
@@ -429,9 +429,7 @@ A great way to see the core functionality in action is to look in the
 Redirecting
 ~~~~~~~~~~~
 
-If you want to redirect the user to another page, use the
-:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::redirect`
```

Comment:
Controller API method links are removed as these methods are protected. Only public methods are included in the online API doc.